test/core/test/dom.test.ts

```diff
@@ -171,19 +171,3 @@ it('doesn\'t throw, if listening for error', () => {
   dispatchEvent(new Event('custom'))
   expect(spy).toHaveBeenCalled()
 })
-
-it('timers are not run after environment teardown', async () => {
```

Comment:
Removing this test case as timers are really not cleared. I think this failed on CI as well at some point.

Btw looking at JSDOM, it claims that it would clear timers when `window.close()` is called: https://github.com/jsdom/jsdom#closing-down-a-jsdom

> use `window.close()`, which will terminate all running timers

I wonder why this doesn't happen in Vitest.

The `happydom` also seems to do same. Vitest is already using its `cancelAsync` https://github.com/capricorn86/happy-dom/blob/13bcfe77ae9a202a93ce8fddcf4a7c1b43ecde16/packages/jest-environment/README.md?plain=1#L124-L125